apps/api/src/database/migrations/1720458965605-AddWebhookTable.ts

```diff
@@ -0,0 +1,63 @@
+import { MigrationInterface, QueryRunner, Table, TableForeignKey } from 'typeorm';
+
+export class AddWebhookTable1720458965605 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(
+      new Table({
+        name: 'notify_webhooks',
+        columns: [
+          {
+            name: 'id',
+            type: 'int',
+            isPrimary: true,
+            isGenerated: true,
+            generationStrategy: 'increment',
+          },
+          {
+            name: 'provider_id',
+            type: 'int',
+          },
+          {
+            name: 'webhookUrl',
+            type: 'varchar',
+            length: '255',
+            isNullable: true,
+          },
+          {
+            name: 'is_verified',
+            type: 'tinyint',
+            default: 0,
+          },
+          {
+            name: 'created_on',
+            type: 'timestamp',
+            default: 'CURRENT_TIMESTAMP',
+          },
+          {
+            name: 'updated_on',
+            type: 'timestamp',
+            default: 'CURRENT_TIMESTAMP',
+            onUpdate: 'CURRENT_TIMESTAMP',
```

Comment:
Add status column with default value as 1.

Reply:
```jsonc

          {
            name: 'status',
            type: 'tinyint',
            default: 1,
          },
```

Reply:
Added this change in migration